CONTRIBUTING.adoc

```diff
@@ -31,7 +31,12 @@ Before we accept a non-trivial patch or pull request we will need you to https:/
 Signing the contributor's agreement does not grant anyone commit rights to the main repository, but it does mean that we can accept your contributions, and you will get an author credit if we do.
 Active contributors might be asked to join the core team, and given the ability to merge pull requests.
 
+== Documentation contributions
 
+The documentation is stored in this repository, in the link:https://github.com/spring-projects/spring-boot/tree/master/spring-boot-project/spring-boot-docs/src/docs/asciidoc[spring-boot-docs] project.
+
+You may use GitHub's built-in editor for Asciidoc by pressing the top-right Edit button to edit the documentation in browser.
```

Comment:
I am not sure we should bother describing the various ways changes can be contributed on GitHub. As a matter of a fact, I prefer contributions that originate from "real branches" as it makes the follow-up easier. We expect contributors to build the documentation and check that the output is valid which you cant' do using the web editor.

Reply:
Aren't there automatic checks (doc build) on a PR request even when using the github's web editor?

Reply:
A contributor is expected to validate their changes before submitting a PR and I am not keen to let contributors use our CI infrastructure to validate their changes.

Part of that is because if you keep this trial and error until you push something that works, you'd trigger a notification for each commit attempt to the 3K people watching this repository.